source/administration/mmctl-cli-tool.rst

```diff
@@ -2056,6 +2102,36 @@ mmctl group team status
 
 **Options inherited from parent commands**
 
+.. code-block:: sh
+
+   --format string               the format of the command output [plain, json] (default "plain")
+   --insecure-sha1-intermediate  allows the use of insecure TLS protocols, such as SHA-1
+   --local                       allows communicating with the server through a unix socket
+   --strict                      will only run commands if the mmctl version matches the server one
+   
+mmctl integrity
+---------------
+
+**Description**
+
+  Perform a relational integrity check which returns information about any orphaned record found.
```

Comment:
Adding local mode only note:

```suggestion
  Perform a relational integrity check which returns information about any orphaned record found. This command can only be run using local mode.
```

Maybe instead of here we could add this as a warning or note consistent through all the local mode only or remote only commands

Reply:
Yes that's a good idea. At this stage it's only ``mmctl user deleteall`` that's also only in local mode so I'll update it in this PR.